src/Microsoft.Health.Dicom.Core/Extensions/DicomTagExtensions.cs

```diff
@@ -22,7 +23,30 @@ public static class DicomTagExtensions
         public static string GetPath(this DicomTag dicomTag)
         {
             EnsureArg.IsNotNull(dicomTag, nameof(dicomTag));
-            return dicomTag.Group.ToString("X4") + dicomTag.Element.ToString("X4");
+            return dicomTag.Group.ToString("X4").ToUpperInvariant() + dicomTag.Element.ToString("X4").ToUpperInvariant();
```

Comment:
>X [](start = 44, length = 1)

Uppercase `"X"` should be sufficient to make the characters uppercase, as per this doc: [https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#the-hexadecimal-x-format-specifier](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#the-hexadecimal-x-format-specifier)